util/dashboard_config.py

```diff
@@ -398,7 +398,11 @@ def prepare_ontologies(ontologies, ontology_dir, dashboard_dir, make_parameters,
         ont_results_path = os.path.join(ont_dashboard_dir, "dashboard.yml")
         ont_results = ontologies_results[o]
 
-        if os.path.exists(ont_results_path) and config.is_skip_existing():
+        if config.is_skip_existing():
+            continue
+        
+        if ont_results.get('changed') is False and os.path.exists(ont_results_path):
```

Comment:
if the result of `ont_results.get('changed')` is a string, its result by be [truthy](https://www.freecodecamp.org/news/truthy-and-falsy-values-in-python/). So weirdly, if you set

```
ont_results['changed']='False'
```

You may get 

`ont_results['changed']` evaluating to true.. This may not matter here, just warning about it

Reply:
how can it be a string if we define that in the pipeline?

Reply:
Just saying, not a biggie :P